pkgs/tools/networking/haproxy/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, pkgs, fetchurl, openssl, zlib }:
+{ stdenv, pkgs, fetchurl, lua5_3, openssl, pcre, zlib }:
```

Comment:
I think it might be nice to have `useLua` and `usePcre` options so this can be customized with an override.